examples/chefapi_examples/recipes/chef_objects.rb

```diff
@@ -1,51 +1,11 @@
 # Add chef objects to the server for testing
 
-execute 'Set the host name' do
-  command 'hostname testhost'
-end
-
 # Create an organization
-
 execute 'create test organization' do
   command '/opt/opscode/bin/chef-server-ctl org-create test test_org'
   not_if '/opt/opscode/bin/chef-server-ctl org-list |grep test'
 end
 
-execute 'get the ssl certificate for the chef server' do
-  command 'knife ssl fetch'
-  not_if { ::File.exist? '/root/.chef/trusted_certs/testhost' }
-end
-
-# Register this node with the server
-directory '/etc/chef'
```

Comment:
I'm not sure why the kitchen instance was being registered to the chef server. It did not seem to be necessary for any of the tests, and I would not anticipate a machine being a node a prerequisite of running `go-chef`.